module/x/peggy/abci.go

```diff
@@ -20,17 +20,16 @@ func slashing(ctx sdk.Context, k keeper.Keeper) {
 	params := k.GetParams(ctx)
 	currentBondedSet := k.StakingKeeper.GetBondedValidatorsByPower(ctx)
 
-	maxHeight := uint64(ctx.BlockHeight()) - params.SignedValsetsWindow + 1
+	maxHeight := uint64(ctx.BlockHeight()) - params.SignedValsetsWindow
```

Comment:
minor nit, assigning this conditionally rather than assigning it and then maybe assigning it again if underflow is detected seems safer. 

It would allow us to someday panic on under/over flow and have this code still work.